qark/vulnerability.py

```diff
@@ -44,3 +45,12 @@ class Severity(Enum):
     WARNING = 1
     ERROR = 2
     VULNERABILITY = 3
+
+
+class VulnerabilityEncoder(JSONEncoder):
+    def default(self, vulnerability):
+        return dict((('severity', vulnerability.severity.name), ('name', vulnerability.issue_name)))
```

Comment:
Since this should be for any object that cannot be serialized already, we should try/except for `AttributeError` (i.e. `vulnerability` is not a Vulnerability), with the except doing something for a different type.

It can be done here or in `vulnerability_json`. I don't think that we would need to `raise SystemExit`, but just log that something incorrect is being passed in.

Reply:
That's a fair point. I can put that in `VulnerabilityEncoder` as I think that's where it best fits.

Reply:
It now checks for an argument of type Issue and will raise a TypeError if it doesn't get one.